e107_plugins/forum/shortcodes/batch/post_shortcodes.php

```diff
@@ -442,17 +442,17 @@ function sc_forum_post_captionname()
 //$this->forumObj->threadGet($this->id, false)		
     if ($this->var['action'] == "rp")
     {
-      $pre = LAN_FORUM_1003;
-			$name = strip_tags($tp->toHTML($this->var['thread_name'], false, 'no_hook, emotes_off'));
-			$url = e107::url('forum', 'topic', $this->var);
-      $post = LAN_FORUM_2006;
+      	$pre = LAN_FORUM_1003;
+	$name = $this->var['thread_name'];
+	$url = e107::url('forum', 'topic', $this->var);
+      	$post = LAN_FORUM_2006;
     }
     if ($this->var['action'] == "nt")
     {
-      $pre = LAN_FORUM_1001;
-			$name = strip_tags($tp->toHTML($this->var['forum_name'], false, 'no_hook, emotes_off'));
```

Comment:
Cameron meant using this 
```php
$tp->toHtml($this->var['forum_name'], false, 'TITLE');
```

So basically just remove the strip_tags() only 
Now, you have also removed the toHtml method. 
Same applies to the thread_name on line 445.